### DISCUSSION TITLE
Ragdoll's still vanish with Medium low config even after making ragdolls on high?

### DISCUSSION THREAD

**Discussion by yourbuggedup** [mastercoms/tf2cfg#578] [Q&A]
THis is a total noob question and I Hope it can be solved and removed from this discussion board as fast as possible but i really like ragdolls cause I think its funny to just see them fly in random dumb ways sometimes.

I used the medium low config and made sure to make all the models and ragdolls as high quailty as possible and I'm getting the FPS goal i want but for some reason the ragdolls seem to be random. like sometimes they'll stay there for a while then randomly vanish.

Sometimes ill kill someone and the ragdoll never drops at all, its very confusing and I tried to reinstall it a few times but still no luck.

Am i missing a setting to enable to a higher setting?

**Comment by wgetJane** [comment]
ragdolls are limited to one per player and there's no way to change it

**Reply by yourbuggedup** [reply]
Is there anyway to take the medium config and get it as close to the medium low?

**Reply by mastercoms** [reply]
You can use the medium low preset if you want medium low. Or, you can look at the medium low config and see what modules to copy from there
